src/components/NewDisbursementButton.tsx

```diff
@@ -2,22 +2,35 @@ import { Button, Icon } from "@stellar/design-system";
 import { useNavigate } from "react-router-dom";
 import { ShowForRoles } from "components/ShowForRoles";
 import { Routes } from "constants/settings";
+import { useCircleAccount } from "hooks/useCircleAccount";
 
-export const NewDisbursementButton = () => {
+export const NewDisbursementButton = ({
+  size = "sm",
+}: {
+  size?: "sm" | "md" | "xs";
+}) => {
   const navigate = useNavigate();
+  const { isCircleAccount, isCircleAccountPending } = useCircleAccount();
 
   const goToNewDisbursement = () => {
     navigate(Routes.DISBURSEMENT_NEW);
   };
 
+  const inActiveCircleAccount = isCircleAccount && isCircleAccountPending;
+  const disableMessage = inActiveCircleAccount
```

Comment:
Should `inActiveCircleAccount` be named `inactiveCircleAccount` instead?

Reply:
Yes, thank you!